packages/client/src/components/share/toolTip.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Tooltip, Icon} from 'antd';
+import {LightA} from 'components/share';
+
+interface Props {
+  tipText: string;
+  tipLink: string;
+  placement?: any;
```

Comment:
Here `placement` types can consider define as:

```suggestion
  placement?: 'top' | 'left' | 'right' | 'bottom' | 'topLeft' | 'topRight' | 'bottomLeft' | 'bottomRight' | 'leftTop' 'leftBottom' | 'rightTop' | 'rightBottom' ;
```

ref: https://ant.design/components/tooltip/#header